app/javascript/elm/src/TimeRange.elm

```diff
@@ -64,7 +65,7 @@ view refreshTimeRange isDisabled tooltipIcon resetIcon =
             , placeholder "time frame"
             , type_ "text"
             , name "time-range"
-            , disabled isDisabled
+            , disabled (status == Active)
```

Comment:
I'm not sure about this. Maybe TmieRange should know the session status, it should only be interested in knowing if it's disabled? Similar to how you did it with `viewToggleButton`.